week-4/implement/count.test.js

```diff
@@ -15,3 +15,31 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+
+function countChar(str, char) {
+    let count = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str.slice(i, i + 1) === char) {
+            count++;
+        }
+    }
+    return count;
+}
+
+
+console.log(countChar("aaaaa", "a")); 
+console.log(countChar("Hello, World!", "o")); 
+console.log(countChar("abc123", "z")); 
```

Comment:
Remember, instead of using `console.log()`, using tests is a much more clear way. The main reason behind this is, whenever you write `console.log()` in your code, it will give unnecessary amount of outputs when running the program. But if you use tests, you can just simply use the `npm test` command to specifically test your program.